Assets/MixedRealityToolkit.Staging/UnityAR/Editor/ConfigurationChecker/UnityARConfigurationChecker.cs

```diff
@@ -15,19 +16,47 @@ public class UnityARConfigurationChecker
     {
         static UnityARConfigurationChecker()
         {
+            EnsureArFoundationDefine();
```

Comment:
After thinking about it...I'm not sure if this entirely fixes #6856. The asmdef still depends on specific assemblies in the ARFoundation package. Does it resolve gracefully when the package isn't installed?

Reply:
Looks like PR validation failed and answered my question: 
![image](https://user-images.githubusercontent.com/3580640/70763304-7c440d80-1d08-11ea-97bd-d0b8e28c94f2.png)

Reply:
I've got a thought on how to address this. Should submit the change today

Reply:
I've got a thought on how to address this. Should submit the change today

Reply:
should be addressed it the latest commit